packages/grafana-runtime/src/services/live.ts

```diff
@@ -20,12 +21,20 @@ export interface LiveDataFilter {
  */
 export interface LiveDataStreamOptions {
   addr: LiveChannelAddress;
-  frame?: DataFrame; // initial results
+  frame?: DataFrameJSON; // initial results
```

Comment:
`options.frame` is passed from the main thread to web worker and `DataFrame` class is not serializable due to Vectors